app/renderer/components/preferences/advancedTab.js

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const React = require('react')
+const ImmutableComponent = require('../immutableComponent')
+const {StyleSheet, css} = require('aphrodite/no-important')
+const commonStyles = require('../styles/commonStyles')
+
+// Actions
+const {getSetting} = require('../../../../js/settings')
+
+// Components
+const {SettingsList, SettingItem, SettingCheckbox} = require('../settings')
+const {SettingDropdown} = require('../dropdown')
+
+// Constants
+const settings = require('../../../../js/constants/settings')
+const {scaleSize} = require('../../../common/constants/toolbarUserInterfaceScale')
+
+// Utils
+const {changeSetting} = require('../../lib/settingsUtil')
+
+class AdvancedTab extends ImmutableComponent {
+  render () {
+    return <div>
+      <main className={css(styles.advancedTabMain)}>
+        <div className='sectionTitle' data-l10n-id='contentSettings' />
```

Comment:
Basically you need to change this line in the same was @luixxiul did it https://github.com/brave/browser-laptop/pull/8237/files#diff-e3eeb751016b2ce9f8278efce585a461L659